packages/components/src/buttons/src/ClearButton.module.css

```diff
@@ -46,38 +56,80 @@
     text-decoration: none;
     white-space: nowrap;
 
-    background-color: var(--background-color);
+    background: none;
     border: none;
     border-radius: var(--hop-ClearButton-border-radius);
     outline: none;
 
-    transition:
-        color var(--transition),
-        background-color var(--transition);
+    transition: color var(--transition);
+}
+
+.hop-ClearButton::before {
+    content: '';
```

Comment:
I think this should be double quotes. Do we have a standard?

Reply:
it should be, I guess css pages don't correct it.